### ISSUE TITLE
Shall we try to communicate EUR/kg CO_2-reduction to the user?

### ISSUE THREAD

**Issue by ChaelKruip** [quintel/etmodel#2635]
There are many routes toward a CO2-reduction but which are the cheapest?

A route to a given reduction of CO2-emissions usually consists of several steps/measures. To assess which measures should be taken is often expressed as a maximum abatement cost plot such as this one:

![maximum_abatement_cost_plot_-_google_search](https://user-images.githubusercontent.com/1303760/34825402-a3da7446-f6d2-11e7-8a89-f51c22df264a.png)

The first measures (left) actually save money! The further you go to the right, the more it costs to reduce the next kg of CO2.

Such plots are usually misleading because the measures are almost never independent. The CO2 reduced by switching to LED lights will be very different if the electricity mix is changed first etc.

Perhaps we could create a similar plot in the ETM which updates dynamically to the measures taken. 

NOTE: It is not guaranteed that taking the steps in the order that the plot indicates results in the cheapest *path*! You would have to evaluate the complete path (final scenario) to be able to see if it results in a cheaper energy system. Also, the ETM does not keep track of 'costs made to get somewhere' as it does not evaluate transition paths... That will be the topic of another discussion ticket soon 😄 

@jorisberkhout @MartLubben @antw @grdw @AlexanderWirtz @marliekeverweij @DorinevanderVlies @michieldenhaan I am curious about your ideas and opinions on this!

**Comment by MartLubben** [comment]
First and short reaction: 

Pro:
- The graph would give a quick overview of (one of) the most important KPI's of the energy transition. At this moment it is not so clear how to save costs. 
- It fits to the quest of minister Wiebes to 'hunt' for the cheapest Mtons. 

Neutral:
- The scope of the costs vs. complexity could get a thing. For example, more wind could mean cheaper hydrogen, more electric cars could mean cheaper storage, and more (short-term) improvement of (fossil based) industrial processes could mean the lack of investments for a system change, resulting in a bankruptcy and even more overall costs. 
- With the increased complexity the modelling of the costs is also going to be more complex. Even the simplest chart could be quite a challenge. 

Con:
- As the result of the remarks made above: I am afraid that this would be quickly too simplified, or too complex.

**Comment by AlexanderWirtz** [comment]
I am mildly sceptical about this as I have never quite understood how such quantifications can work or be meaningful _per measure_. I see how one might quantify the costs of CO2 mitigation for the total package, but how do you assign costs to individual measures and if you do so straightforwardly (i.e. only taking first order effects into account) how do you communicate the essential insight that it is often combinations of measures / developments that determine the efficacy and therefore costs-effectiveness.

In short, you yourself acknowledge that there is little independence, so how do we get around this problem. Please, prove me wrong and short-sided.

**Comment by ChaelKruip** [comment]
> Please, prove me wrong and short-sided.

I don't think you are either 😄 
One idea that I had which is not very practical but uses the fact that the ETM 'knows' all these possibly complex interdependencies:

For each measure that you want to evaluate (in the context of the scenario that you are in), move the slider for that measure a tiny step up/down and register the change in total costs and the CO2 reduction. To build a chart with 10 measures, you would have to move 10 sliders (and return them to the current scenario position).

That way, the MAC plot represents the 'marginal costs' of reducing CO2 in a way that does justice to the scenario that you have constructed...
